docs/release_notes.md

```diff
@@ -29,11 +34,11 @@ Recreate stack exchange redis connection multiplexer, if it goes into a bad stat
 https://github.com/Azure/aspnet-redis-providers/issues/69
 
 ### v2.2.4
-Lazy deserialize of session data: Deserialize session data only when accessed by application and not when fetched from Redis. As well as, Updated StackExchange.Redis.   
+Lazy deserialize of session data: Deserialize session data only when accessed by application and not when fetched from Redis. As well as, Updated [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/).   
 Issues fixed: https://github.com/Azure/aspnet-redis-providers/issues/46  
```

Comment:
For SE.Redis, I'd link to the repo at `https://github.com/StackExchange/StackExchange.Redis` rather than the docs at `https://stackexchange.github.io/StackExchange.Redis/`